src/main/java/api/apiwrappers/OrderApiWrapper.java

```diff
@@ -14,9 +14,32 @@ public Order postOrder(Order order) throws IOException {
     }
 
     public Order getOrder(Order order) throws IOException {
-        String endpoint = baseUrl + "/v2/store/order/" + order.getId().toString();
+        String endpoint = baseUrl + "/v2/store/order/" + order.getId();
         
         return (Order)get(order, endpoint);
     }
 
+    public Order getOrderInventory(Order order) throws IOException {
```

Comment:
Same as at Pet class, handle the exceptions in this class.

Reply:
Added TryCatch exception